EIPS/eip-4626.md

```diff
@@ -45,21 +46,28 @@ which may affect precision for computing the value of Vault shares
 ERC-4626 MAY implement [EIP-2612](./eip-2612.md) to improve the UX of approving shares on various integrations.
 
 ### Definitions:
+
 - asset: The underlying token managed by the Vault.
-         Has units defined by the corresponding ERC20 contract.
+  Has units defined by the corresponding ERC20 contract.
+
 - share: The token of the Vault. Has a ratio of underlying assets
-         exchanged on mint/deposit/withdraw/redeem (defined by the Vault).
+  exchanged on mint/deposit/withdraw/redeem (defined by the Vault).
+
 - slippage: Any difference between advertised share price and economic realities of
-            deposit to or withdrawal from the Vault, which is not accounted by fees.
-- fee: An amount of assets or shares charged to the Vault. Can be a fee on deposits, yield, AUM, withdrawal, or any other kind of fee.
+  deposit to or withdrawal from the Vault, which is not accounted by fees.
+
+- fee: An amount of assets or shares charged to the Vault. Can be a fee on deposits,
+  yield, AUM, withdrawal, or any other kind of fee.
```

Comment:
Besides the extra spacing between the defs, I don't think this indentation changes anything in how it's displayed